doc/whatsnew/fragments/7626.false_positive

```diff
@@ -0,0 +1,6 @@
+Fix a false positive for ``simplify-boolean-expression`` when multiple values
+are inferred for a constant.
+
+Adds a keyword-only ``compare_constants`` argument to ``safe_infer``.
```

Comment:
This one is an internal change, but we might want to add two fragments as it's indeed valuable to know for plugin developers.
```suggestion
```